Python/Product/Analysis/LanguageServer/Server.Completion.cs

```diff
@@ -57,7 +58,7 @@ public sealed partial class Server {
                 members = members.Where(m => m.kind == filterKind.Value);
             }
 
-            var res = new CompletionList {
+             var res = new CompletionList {
```

Comment:
Why the extra space?

Reply:
Accident. Most prob VS Code C# formatter, was working on Mac.